platform/logging/elasticsearch-cluster-master.yaml

```diff
@@ -27,9 +27,13 @@ spec:
       master: "true"
       ingest: "false"
       data: "false"
+    persistence:
+      enabled: true
+      annotations:
+        helm.sh/resource-policy: delete
```

Comment:
Don't use this annotation since we are already declaring the reclaimPolicy in storageclass